main/system.h

```diff
@@ -4,6 +4,18 @@
 #include "esp_err.h"
 #include "global_state.h"
 
+#ifdef CONFIG_STRATUM_EXTRANONCE_SUBSCRIBE
+    #define STRATUM_EXTRANONCE_SUBSCRIBE 1
```

Comment:
any reason to rdefine the config like this? Can we not just have the config default to 0?

Reply:
If it's 0, the config is not in `sdkconfig.h`. Why, I don't know.